client/src/CodeClaimPage/index.tsx

```diff
@@ -47,78 +52,92 @@ export const CodeClaimPage: React.FC<RouteComponentProps<{ hash: string, method:
 
   useEffect(() => {
     if (claim && claim.delegated_mint && !isVerified) {
-      verifySignedMessage()
+      verifySignedMessage();
     }
   }, [claim]); /* eslint-disable-line react-hooks/exhaustive-deps */
 
   const fetchClaim = (hash: string) => {
     setIsClaimLoading(true);
     getClaimHash(hash.toLowerCase())
-      .then(claim => {
+      .then((claim) => {
         setClaim(claim);
         setClaimError(false);
       })
-      .catch(error => {
+      .catch((error) => {
         setClaimError(true);
       })
       .finally(() => setIsClaimLoading(false));
-  }
+  };
 
   const continueClaim = (claim: HashClaim) => {
-    setClaim(claim)
-    setInitialStep(false)
-  }
+    setClaim(claim);
+    setInitialStep(false);
+  };
 
   const checkUserTokens = () => {
-    if (!claim || !claim.beneficiary) return
-    getTokensFor(claim.beneficiary)
-      .then(tokens => {
-        if(tokens.filter(token => token.event.id === claim.event_id).length > 0) {
-          setBeneficiaryHasToken(true)
-        }
-      })
-  }
+    if (!claim || !claim.beneficiary) return;
+    getTokensFor(claim.beneficiary).then((tokens) => {
+      if (tokens.filter((token) => token.event.id === claim.event_id).length > 0) {
```

Comment:
Can we improve this a little bit?

```suggestion
const hasTokens = tokens.filter((token) => token.event.id === claim.event_id)).length > 0;
      if (hasTokens) {
```